pxtblocks/blocklylayout.ts

```diff
@@ -119,7 +119,14 @@ namespace pxt.blocks.layout {
             cvs.width = width * pixelDensity;
             cvs.height = height * pixelDensity;
             img.onload = function () {
-                ctx.drawImage(img, 0, 0, width, height, 0, 0, cvs.width, cvs.height);
+                try {
+                    ctx.drawImage(img, 0, 0, width, height, 0, 0, cvs.width, cvs.height);
```

Comment:
is this due to incorrect SVG or security?

Reply:
It's a browser exception (`IndexSizeError`). I've updated the PR to avoid this codepath in IE entirely since this fix was only meant for "download screenshot" and there's no point in catching it.